### PR TITLE
[Misc] Update version

### PR DESCRIPTION
**Pull request by Accelerator1996**

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-8.422.8.20.21.01-GA
\ No newline at end of file
+8.432.8.23.22.01-GA
```
Comment (Accelerator1996) [inline version.txt:1]: diff --git a/dragonwell_version b/dragonwell_version
index 1042bc707d..5a2dfbc2c8 100644
--- a/dragonwell_version
+++ b/dragonwell_version
@@ -19,5 +19,5 @@
 # Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-DRAGONWELL_VERSION=8.23.22
-DRAGONWELL_JDK_UPDATE_VERSION=432
+DRAGONWELL_VERSION=8.20.21
+DRAGONWELL_JDK_UPDATE_VERSION=422
diff --git a/version.txt b/version.txt
index 610a8b4144..99581650d7 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-8.432.8.23.22.01-GA
\ No newline at end of file
+8.422.8.20.21.01-GA
\ No newline at end of file

Comment (Accelerator1996) [inline version.txt:1]:
```
diff --git a/dragonwell_version b/dragonwell_version
index 5a2dfbc2c8..1042bc707d 100644
--- a/dragonwell_version
+++ b/dragonwell_version
@@ -19,5 +19,5 @@
 # Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-DRAGONWELL_VERSION=8.20.21
-DRAGONWELL_JDK_UPDATE_VERSION=422
+DRAGONWELL_VERSION=8.23.22
+DRAGONWELL_JDK_UPDATE_VERSION=432
diff --git a/version.txt b/version.txt
index 99581650d7..610a8b4144 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-8.422.8.20.21.01-GA
\ No newline at end of file
+8.432.8.23.22.01-GA
\ No newline at end of file
```

**Review by kuaiwei — APPROVED** [review]
LGTM

**Review by sendaoYan — APPROVED** [review]
/LGTM

**Pull request merged**